frontend/src/components/eMIB/EmibTabs.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import HowTo from "./HowTo";
+import Background from "./Background";
+import Inbox from "./Inbox";
+import { SELECTED, UNSELECTED } from "../common/Tab";
+import LOCALIZE from "../../text_resources";
+import TabNavigation from "../common/TabNavigation";
+
+class EmibTabs extends Component {
+  render() {
+    let TABS = [
```

Comment:
`const` instead of `let` because this never changes.

Reply:
changed